### DISCUSSION TITLE
Can I simulate nano antenna in MEEP

### DISCUSSION THREAD

**Discussion by VyshnuP** [NanoComp/meep#2897] [Q&A] [answered]
I am an undergrad student who wants do design a metamaterial based nano antenna, I've been searching for a lot of software but none were apt for designing nano-antenna as students versions are limited. I came across MEEP and want to know if I can simulate this or are there any other software or tools for me to use. Please I need guidance as I have limited knowledge on this.

**Comment by stevengj** [comment, marked as answer]
Yes.  The caveat is that FDTD methods like Meep with a uniform spatial grid are fairly inefficient for modeling plasmonic metals, since they need very high spatial resolution to resolve the skin depth and then wastefully use that same resolution even in the air regions.  Finite-element methods (FEM) can be more efficient for this, and there are a variety of free FEM packages out there.   That being said, people definitely use FDTD for this kind of thing too.   e.g. this is the first paper that comes up in a google search for "FDTD nano-antenna", and they describe how they use FDTD to simulate a nano-antenna: https://www.nature.com/articles/s41598-019-49517-y

**Reply by VyshnuP** [reply]
Thankyou very much, I never thought of doing them using FEM method. IF you wouldn't mind can you suggest the tools for me to use for designing this. And thankyou for the reference paper too
